appdaemontestframework/appdaemon_mock/appdaemon.py

```diff
@@ -0,0 +1,18 @@
+import asyncio
+import pytz
+
+class AppDaemon:
```

Comment:
Would you mind renaming to `MockAppDaemon` to make more clear what this actually is. 🙂